mw-webapp/src/utils/DateUtils.ts

```diff
@@ -23,7 +23,14 @@ export class DateUtils {
     const currentDate = new Date();
     const dateInPast = new Date(currentDate.getTime() - (amount * DAY_MILLISECONDS));
 
-    return new Date(dateInPast);
+    return new Date(dateInPast.toDateString());
+  }
+
+  /**
+   * Change date to date yyyy-mm-dd
+   */
+  public static getToDate(date: Date): Date {
+    return new Date (date.toDateString());
```

Comment:
As I see the method returns not yyyy-mm-dd, but Date object. Don't be so fast. Take a few minutes to describe utils properly. They could be highly reusable

Reply:
Lets rename: getToDate -> roundToDate

Reply:
Done